lib/vendor/EmblaCarousel/Carousel/Carousel.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import {
+  DotButton,
+  PrevButton,
+  NextButton,
+} from '../CarouselButtons/CarouselButtons';
+import useEmblaCarousel from 'embla-carousel-react';
+import './carousel.module.css';
+
+const Carousel = ({ children }) => {
+  const [scrollSnaps, setScrollSnaps] = useState([]);
+  const [viewportRef, embla] = useEmblaCarousel({ loop: false });
+  const [prevBtnEnabled, setPrevBtnEnabled] = useState(false);
```

Comment:
Let's make this cleaner by calling them `isPreviousButtonEnabled` and `setIsPreviousButtonEnabled`.